### ISSUE TITLE
Smarter ViewController Identification

### ISSUE THREAD

**Issue by sirnacnud** [k0nserv/luft#21]
I have some test case classes that have ViewController in the name that are being picked up as false positives by Luft. I was wondering if there were some improvements we could do to eliminate them be identified as ViewControllers by Luft.

For my codebase:
-  ViewController classes always end in `ViewController` 
-  Test case classes always end in `Tests`

I'm these were common naming schemes, we could modify Luft to better determine if the class is actually a ViewController.

Another thought I had was if it were possible to use Xcode to somehow determine if a ViewController is in the inheritance hierarchy of the class. I think this would involve parsing the class to determine it's inheritance. I poked around the Xcode headers a bit, but I'm not for sure if it could be easily done.

**Comment by bogren** [comment]
Something like this is going to work for **all** my projects :point_down:  (Swift and Object :+1:)

```
- (BOOL)isViewController:(NSString *)filename {
...
NSString *lowerCaseFilename = [filename lowercaseString];
BOOL isTest = [lowerCaseFilename rangeOfString:@"test"].location == NSNotFound;
...
return isViewController && isImplementationOrSwift && isTest;
```

I tried to find something else that differ tests form implementation files, it's hard when there is mix of Swift and Objc.

**Comment by k0nserv** [comment]
Absolutely. I think something `*ViewController" would be good. It wouldn't match`Test` at the end

**Comment by sirnacnud** [comment]
Alright, something I just thought about was the case of categories, where you have something like `ViewController+Foo`. Right now they are being identified as a ViewController by Luft. If we change Luft so `ViewController` is at the end, categories won't be matched. I don't know if categories should be included or not, at least for my code base categories are typically small extensions to a class.

**Comment by k0nserv** [comment]
Yeah seems more reasonable to not include categories. A regex on the form `.*ViewController$` seems like the best option imo

**Comment by sirnacnud** [comment]
Alright, I'll work on a PR for it then

**Comment by k0nserv** [comment]
I'll make you a collaborator. Please do submit a PR though so we can review it together

**Comment by sirnacnud** [comment]
Created pull request #22

**Comment by k0nserv** [comment]
Fixed with #22
